apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.story.jsx

```diff
@@ -5,27 +5,36 @@ const multipleChoiceData = [
   {
     id: 1,
     question: '1. What is a variable?',
-    percentAnsweredOptionOne: '80%',
-    percentAnsweredOptionTwo: '15%',
-    optionOneIsCorrect: true,
-    notAnswered: '5%',
+    answers: [{},{percentAnswered: '40', isCorrectAnswer: true},
```

Comment:
@caleybrock This is a hack to ensure that the answers' columns start from column 1.  Any suggestions will be appreciated?  I wonder if I should add  + 1 to my column build out.